README.md

```diff
@@ -249,6 +256,22 @@ postgresql_database 'vacuum databases' do
 end
 ```
 
+```ruby
+# Create, Insert, Query a SQLite database
+# Note that inserting anything in to the database will create it automaticly.
+sqlite_database 'mr_softie' do
+  connection '/path/to/database.db3'
```

Comment:
```database_name``` seems to be the common database name object, postgresql allows the ```database``` part in the connection hash to override database_name. as such i think that database_name is the better object to use. I'll make the adjustment to database_name, unless you have strong objections.